doc/lightning_tensor/device.rst

```diff
@@ -0,0 +1,184 @@
+Lightning Tensor device
+=======================
+
+The ``lightning.tensor`` device is a tensor network simulator device. The device is built on top of the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`__ from the NVIDIA cuQuantum SDK, enabling GPU-accelerated simulation of quantum tensor network evolution.
+
+A ``lightning.tensor`` device with the default setup can be simply loaded using:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.tensor", wires=100)
+
+By default, the device employs the ``Matrix Product State (MPS)`` tensor network approximation to represent the quantum state. 
+Within this framework, the maximum bond dimension (``max_bond_dim``) defaults to ``128``. To manage computational complexity, 
+singular value truncation is utilized, with the default threshold for truncation (``cutoff``) set at ``0``. Additionally, the 
+singular value truncation mode (``cutoff_mode``) defaults to ``abs``, considering the absolute values of the singular values.
+Alternatively, users can opt to set ``cutoff_mode`` to ``rel`` to consider the relative values of the singular values.
```

Comment:
The info provided is great! I would find it more accessible if they are listed in bullet points as (keyword) arguments as is typically done in code documentation. This is not a strict requirement, I just personally find walls of text in docs off-putting (subjective)